app/models/expense.rb

```diff
@@ -0,0 +1,5 @@
+class Expense < ApplicationRecord
+  validates :title, presence: true
+  validates :value, presence: true,
+            numericality: { greater_than: 0 }
```

Comment:
It's a good habit to align hash values to the same level. Align `numericality` to `presence` please :).